crossbar/twisted/endpoint.py

```diff
@@ -213,6 +213,12 @@ def _create_tls_server_context(config, cbdir, log):
 
     # create a TLS context factory
     # see: https://twistedmatrix.com/documents/current/api/twisted.internet.ssl.CertificateOptions.html
+    # XXX can we get CertificateOptions to do this for us?
+    from twisted.web import http
+    protocols = [b'http/1.1']
+    if http.H2_ENABLED:
+        protocols = [b'h2', b'http/1.1']
```

Comment:
Okay, good! I was hoping that was wrong, but couldn't get things to work otherwise. I'll look into the factory interface. (And yes, we're doing a bunch of custom stuff in here so that's probably it).

Thanks for the feedback :)